### ISSUE TITLE
Animations and direction bug

### ISSUE THREAD

**Issue by Yanrishatum** [HeapsIO/heaps#402]
This model seem to load animation incorrectly, when exported via Maya, resulting in broken animations. Other software that can read FBX have no issues with it, source maya file included.
For some reason, converting it to binary (via Autodesk FBX converter), importing to blender, without modifications exporting back to binary and converitng to ascii produces correct animations?
It also loads it with incorrect axes, because it appears to look up, instead of forward. Export was configured with Z axis up and other software loads it correctly as well.
https://drive.google.com/open?id=1AS05xBkYDbxjxeP2hOK2xjuDzMW5YMux

**Comment by ncannasse** [comment]
Yes we have some issues with Maya Axis, as exporting Z-up does actually
apply a rotation to all Root models instead of a properly switching the
axes.
We have a "TODO  axisFlip" in hxd/fmt/fbx/BaseLibrary (updateModelScale) to
handle this correctly.
